src/cat/renderChart.js

```diff
@@ -7,12 +7,12 @@ export function renderChart(cat) {
   cat.settings.sync(cat);
   //render the new chart with the current settings
   var dataFile = cat.controls.dataFileSelect.node().value;
-  var dataFilePath = cat.config.dataURL + dataFile;
+  var dataObject = cat.config.dataFiles.find(f => f.label == dataFile);
```

Comment:
add this polyfill for find to support IE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Polyfill

Reply:
Fixed in `hotfixes` branch.